packages/gatsby-theme-docs/src/components/theme-provider.js

```diff
@@ -27,13 +28,15 @@ const ThemeProvider = (props) => {
     siteMetadata: { ...data.site.siteMetadata, ...additionalData },
   };
   return (
-    <ErrorBoundary>
-      <SiteDataContext.Provider value={siteData}>
-        <Reset />
-        <Globals />
-        {props.children}
-      </SiteDataContext.Provider>
-    </ErrorBoundary>
+    <IntlProvider locale="en">
```

Comment:
For now, since we only have one use case, I would render this in the release notes filters only, not for the entire website.